common/set_benchmark.cpp

```diff
@@ -365,10 +365,23 @@ static void BM_SetInsertSeq(benchmark::State& state) {
     // While this count is "iteration invariant" (it should be exactly the same
     // for every iteration as the set of keys is the same), we don't use that
     // because it will scale this by the number of iterations. We want to
-    // display the probe count of this benchmark *parameter*, not the probe
-    // count that resulted from the number of iterations. That means we use the
-    // normal counter API without flags.
-    state.counters["Probed"] = s.CountProbedKeys();
+    // display the metrics for this benchmark *parameter*, not what resulted
+    // from the number of iterations. That means we use the normal counter API
+    // without flags.
+    auto metrics = s.GetMetrics();
+    state.counters["P-compares"] = metrics.probe_avg_compares;
```

Comment:
Is it worth `CHECK`ing that `metrics.key_count == keys.size()` here?

Reply:
Good idea, done.